src/auth/oauth2/core/qgsauthoauth2method.cpp

```diff
@@ -70,6 +70,7 @@ QgsOAuth2Factory *QgsOAuth2Factory::instance()
   if ( !sInstance )
   {
     sInstance = new QgsOAuth2Factory();
+    qRegisterMetaType<QNetworkReply::NetworkError>( "QNetworkReply::NetworkError" );
```

Comment:
We can't backport the move to QgsApplication?

Reply:
> We can't backport the move to QgsApplication?

the registerMetaTypes() method doesn't exist in that branch

Reply:
ok, fair enough!

Reply:
Actually it should be moved here  - https://github.com/qgis/QGIS/blob/release-3_40/src/core/qgsapplication.cpp#L262